using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -658,22 +761,25 @@ ACPCore.TrackState("State Name", contextData);
 {% endtab %}
 {% endtabs %}
 
-The following video shows you how to use [trackState](../../foundation-extensions/mobile-core/mobile-core-api-reference.md#track-app-states-and-views) APIs to send data to Adobe Analytics.
+The following video shows you how to use [trackState](../../foundation-extensions/mobile-core/mobile-core-api-reference.md#trackstate) APIs to send data to Adobe Analytics. 
+
 
 {% embed url="https://video.tv.adobe.com/v/26260/?quality=12" caption="" %}
+{% hint style="info" %} Video is demonstrated with ACP2.x, latest iOS version is AEP 3.x. {% endhint %}
```

Comment:
Should there be a space between ACP and 2.x?
"ACP2.x" -> "ACP 2.x"
Also, what do you think about adding emphasis on the difference between the SDKs (video vs latest) by updating to: "Video is demonstrated with ACP 2.x, however, the latest iOS SDK version is AEP 3.x."?

Reply:
I agree with your phrasing, it sounds natural and has the emphasis for video vs latest SDK version